packages/server/lib/modes/record.js

```diff
@@ -338,7 +345,8 @@ const uploadArtifactBatch = async (artifacts, protocolManager, quiet) => {
       return skipped && !report.error ? acc : {
         ...acc,
         [key]: {
-          ...report,
+          // TODO: once cloud supports reporting duration, no longer omit this
+          ..._.omit(report, 'duration'),
```

Comment:
Is the Cloud not able to handle extra fields?

Reply:
No, it will give zod errors.